securedrop_client/logic.py

```diff
@@ -85,7 +85,8 @@ class Client(QObject):
 
     finish_api_call = pyqtSignal()  # Acknowledges reciept of an API call.
 
-    def __init__(self, hostname, gui, session, home: str) -> None:
+    def __init__(self, hostname, gui, session,
+                 home: str, proxy: bool = False) -> None:
```

Comment:
`proxy` should be True if the default application is Qubes first

Reply:
Okay.